Identifiable.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//  CoconutBits
+//
+//  Created by Dino on 28/11/2018.
+//
+
+import Foundation
+
+protocol Identifiable {
+    static var identifier: String { get }
+}
+
+extension Identifiable {
+    static var identifier: String {
```

Comment:
@d-srd public + default extension for UIView

Reply:
fixed!